api/app/graphql/types/query_type.rb

```diff
@@ -0,0 +1,22 @@
+module Types
+  class QueryType < Types::BaseObject
+    # Add `node(id: ID!) and `nodes(ids: [ID!]!)`
+    include GraphQL::Types::Relay::HasNodeField
+    include GraphQL::Types::Relay::HasNodesField
+
+    # Add root-level fields here.
+    # They will be entry points for queries on your schema.
+
+    # TODO: remove me
+    field :test_field, String, null: false,
+      description: "An example field added by the generator"
+    def test_field
+      "Hello World!"
+    end
+
+    field :users, [Types::UserType], null: true 
+    def users(**args)
+      User.all
```

Comment:
```suggestion
      ::User.all
```

こんな感じで記述するとどうですか？
もしかすると名前空間の衝突が起きているのかもしれません

Reply:
コメントありがとうございます
試してみたところエラーが"uninitialized constant Types::QueryType::User"から"uninitialized constant User"に変わりました。